libsolidity/CompilerStack.h

```diff
@@ -26,11 +26,18 @@
 #include <ostream>
 #include <string>
 #include <memory>
+#include <vector>
 #include <boost/noncopyable.hpp>
 #include <libdevcore/Common.h>
 #include <libdevcore/FixedHash.h>
 
 namespace dev {
+
+namespace eth {
+	class AssemblyItem;
```

Comment:
It think coding style says no indentation for namespace (and `{` always on a line of its own, so it would be great if you could also correct the errors that are already present in this file :-) ).